### DISCUSSION TITLE
QOL: Withdraw Potions from Potion Storage from a Bank Tag

### DISCUSSION THREAD

**Discussion by TheDaveyBoy** [runelite/runelite#18443] [Ideas]
![image](https://github.com/user-attachments/assets/6cc52516-8f56-42db-86c3-63501699a43e)

**Comment by TheDaveyBoy** [comment]
Potions in the bank tabs can be withdrawn if they are in the Bank, however they cannot be if they are in potion storage.

**Comment by geheur** [comment]
Potion storage is available in the core runelite version of layouts. There is an option in bank tag layouts to show the options to turn it on, then you can export the layout, delete the layout, enable runelite's version, and import the layout.

**Reply by TheDaveyBoy** [reply]
This is amazing, thank you so much!
